package.json

```diff
@@ -37,7 +37,8 @@
     "lodash": "^4.17.15",
     "mime-types": "^2.1.26",
     "serverless-http": "^2.3.1",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "@types/express-serve-static-core": "^4.17.2"
```

Comment:
This is a dev dependency

Reply:
I thought so too, but when I added it as a devDependency it still failed on yarn install. When I moved it to dependencies, it didn't fail anymore. Am I missing a step?

Reply:
Whoops your right: https://stackoverflow.com/a/46011417/9041033

Reply:
I would suggest explicitly calling out the return type here: https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/app.ts#L23